docs/Installation/Installation.md

```diff
@@ -174,19 +174,20 @@ Notes:
 
 ## Using Singularity to obtain a SpECTRE environment
 
-[Singularity](http://singularity.lbl.gov/index.html) is a container alternative
+[Singularity](https://sylabs.io) is a container alternative
 to Docker with better security and nicer integration.
 
 To use Singularity you must:
 
-1. Build [Singularity](http://singularity.lbl.gov/index.html) and add it to your
+1. Build [Singularity](https://sylabs.io) and add it to your
    `$PATH`
 2. `cd` to the directory where you want to store the SpECTRE Singularity image,
    source, and build directories, let's call it WORKDIR. The WORKDIR must be
    somewhere in your home directory. If this does not work for you, follow the
    Singularity instructions on setting up additional [bind
-   points](http://singularity.lbl.gov/docs-mount). Once inside the WORKDIR,
-   clone SpECTRE into `WORKDIR/SPECTRE_ROOT`.
+   points](https://sylabs.io/guides/3.7/user-guide/bind_paths_and_mounts.html) 
+   (version 3.7. For other versions, see the [docs](https://sylabs.io/docs/)). 
```

Comment:
Did this change significantly in version 3.7? Do you think it's necessary to give this version-specific information in _our_ docs or can we rely on _their_ docs to point out these version differences? I would prefer the latter.

Reply:
No nothing changed significantly in version 3.7. I wanted to link to a “stable” or “current” version of their docs for this specific topic, but they only have docs for each version release. So since we reference a specific topic, I just picked the most recent version as of right now to link to. That’s why I included the second link which just goes to their main docs page with all of their versions. I agree it’d be better to not link to a specific version though.

Reply:
I see. Well then I think what you did makes sense 👍